packages/salesforcedx-vscode-apex/package.json

```diff
@@ -2,8 +2,7 @@
   "preview": true,
   "name": "salesforcedx-vscode-apex",
   "displayName": "Apex Code Editor for Visual Studio Code",
-  "description":
```

Comment:
tslint has the one-line rule set to true so I think someone's linter isn't set up properly.

Reply:
What does prettier formatter say? I don't think ts-lint should be running on the .json file. It is not running for me.